Cannot find module @svelte-drama\swr\_memoize

Hello, when I try to use `refreshInterval` it says:

Cannot find module 'XXXXX\node_modules\@svelte-drama\swr\_memoize' imported from XXXXX\node_modules\@svelte-drama\swr\_plugins\refresh-interval.js

Svelte version: ^3.48.0 (latest)

--- a/src/lib/_plugins/refresh-interval.ts
+++ b/src/lib/_plugins/refresh-interval.ts
@@ -1,5 +1,5 @@
 import type { SWRPlugin } from '../types.js'
-import { memoize } from '../_memoize'
+import { memoize } from '../_memoize.js'
 import { derived, readable } from 'svelte/store'
 
 const createTimer = memoize(() => {

